tests/dm_syncer/conf/dm-syncer-1.toml

```diff
@@ -0,0 +1,124 @@
+# Base Configuration.
+
+# sub task name as identifier, should be unique
+name = "hanfei"
+
+# sub task mode, can be `all`, `full`, `incremental`
+mode = "incremental"
```

Comment:
can we set mode = "incremental" in code，because this mode is always incremental if use dm-syncer?

Reply:
done